src/silo/config/config_repository.cpp

```diff
@@ -80,11 +80,10 @@ void validatePartitionBy(const DatabaseConfig& config) {
 
    const std::string partition_by = config.schema.partition_by.value();
 
-   const auto& partition_by_metadata = std::find_if(
-      config.schema.metadata.begin(),
-      config.schema.metadata.end(),
-      [&](const DatabaseMetadata& metadata) { return metadata.name == partition_by; }
-   );
+   const auto& partition_by_metadata =
+      std::ranges::find_if(config.schema.metadata, [&](const DatabaseMetadata& metadata) {
+         return metadata.name == partition_by;
+      });
```

Comment:
```
#9 16.32 /src/src/silo/config/config_repository.cpp: In function ‘void silo::config::{anonymous}::validatePartitionBy(const silo::config::DatabaseConfig&)’:
#9 16.32 /src/src/silo/config/config_repository.cpp:84:20: error: ‘find_if’ is not a member of ‘std::ranges’; did you mean ‘std::find_if’?
#9 16.32    84 |       std::ranges::find_if(config.schema.metadata, [&](const DatabaseMetadata& metadata) {
```

Reply:
Is this local for you or on the server? If local: what is your gcc version?

Reply:
Ah, I see, it is in the CI